x-pack/test/api_integration/apis/apm/agent_configuration.ts

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function featureControlsTests({ getService }: FtrProviderContext) {
```

Comment:
should this function always be called `featureControlsTests`?

Reply:
No, will rename